tests/langchain/test_langchain_model_export.py

```diff
@@ -82,7 +82,8 @@ def _llm_type(self) -> str:
         """Return type of llm."""
         return "fake"
 
-    def _call(self, prompt: str, stop: Optional[List[str]] = None) -> str:
+    # pylint: disable=arguments-differ
+    def _call(self, prompt: str, stop: Optional[List[str]] = None, run_manager=None) -> str:
```

Comment:
Add missing `run_manager` arg but disable this warning for future extensions to `_call` since LangChain APIs change every few days :)